node/request.js

```diff
@@ -43,7 +44,7 @@ function TChannelRequest(channel, options) {
     self.options = options;
     self.triedRemoteAddrs = {};
     self.outReqs = [];
-    self.timeout = self.options.timeout;
+    self.timeout = self.options.timeout || DEFAULT_TIMEOUT;
```

Comment:
sure, this is a dead branch anyhow at present, just added for defense in depth, will update